packetbeat/docs/fields.asciidoc

```diff
@@ -10040,7 +10087,7 @@ type: keyword
 
 --
 
-*`tls.server_certificate.fingerprint.sha256`*::
+*`tls.detailed.server_certificate.fingerprint.sha256`*::
```

Comment:
Should be in `tls.server.hash.sha256` and copied to `related.hash`

Reply:
Field removed.